src/main/java/com/alapmistry/spring_boot_demo/controllers/HealthController.java

```diff
@@ -8,6 +8,7 @@ public class HealthController {
 
     @GetMapping("/health")
     public String healthCheck() {
+        System.out.print("Healthy!");
```

Comment:
Consider using a proper logging framework instead of System.out.print for better control over logging levels and output formatting. **Co-authored by Cerebras AI**